lib/tlIO/CMakeLists.txt

```diff
@@ -67,11 +71,23 @@ if(TLRENDER_FFMPEG)
     list(APPEND SOURCE FFmpeg.cpp FFmpegRead.cpp FFmpegReadAudio.cpp
         FFmpegReadVideo.cpp FFmpegWrite.cpp)
     list(APPEND LIBRARIES_PRIVATE FFmpeg)
+    if(WIN32)
+	set_source_files_properties(FFmpegWrite.cpp
+	    PROPERTIES COMPILE_FLAGS -std:c++20)
+    endif()
 endif()
 if(TLRENDER_USD)
     list(APPEND HEADERS_PRIVATE USD.h USDPrivate.h)
     list(APPEND SOURCE USD.cpp USDRead.cpp USDRender.cpp)
     list(APPEND LIBRARIES_PRIVATE tf usd usdGeom usdImagingGL usdAppUtils hd)
+    if(NOT WIN32)
+	set_source_files_properties(USD.cpp
```

Comment:
Please no not include unrelated changes in the PR.

Reply:
Removed.